doc/Hammer-Use/Hammer-Config.rst

```diff
@@ -226,6 +226,100 @@ The file should contain the same keys as the corresponding configuration file, b
 
 HAMMER will perform the same without a types file, but it is highly recommended to ensure type safety of any future plugins.
 
+Key History
+-----------
+
+When the ``ruamel.yaml`` package is installed, HAMMER can emit what files have modified any configuration keys in YAML format.
+The file is named ``{action}-output-history.yml`` and is located in the output folder of the given action.
+
+Example with the file ``test-config.yml``:
+
+  .. code-block:: yaml
+
+    synthesis.inputs:
+        input_files: ["foo", "bar"]
```

Comment:
Small nit: Files should have `.v` extension and top_module should not have `.xdc` extension (just in case anyone gets confused)